crates/modelardb_server/src/storage/uncompressed_data_manager.rs

```diff
@@ -624,12 +625,12 @@ mod tests {
     use datafusion::arrow::array::StringBuilder;
     use datafusion::arrow::datatypes::SchemaRef;
     use datafusion::arrow::record_batch::RecordBatch;
+    use modelardb_common::metadata::try_new_sqlite_table_metadata_manager;
```

Comment:
Maybe only import `modelardb_common::metadata` and use the function as  `metadata::try_new_sqlite_table_metadata_manager()` for consistency.

Reply:
Fixed.